### PR TITLE
Import images from unsplash API

### PR DESCRIPTION
**Pull request by ankitrox**
Import images from unsplash API

### CODE REVIEW

File: mu-plugins/media-import.php [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+/**
+ * Import the images from unsplash API.
+ *
+ * @return void
+ */
+function webp_analysis_import_images( $args, $assoc_args ) {
+    $endpoint = 'https://api.unsplash.com';
+    $search_photos_path = '/search/photos';
+    $list_photos_path = '/photos';
+    $access_key = 'z3pOnmimIPdiUn8qdr31nPuIFArkLKbz4sfVTw9sXno';
```
Comment (jjgrainger) [inline mu-plugins/media-import.php]: @ankitrox do we need to handle this differently? Not sure if this was waiting on .env support? #3

**Review by jjgrainger — CHANGES_REQUESTED** [review]
Just one thing to address otherwise looks good

**Review by jjgrainger — APPROVED** [review]

File: mu-plugins/media-import.php (hunk 1 above)
Comment (ankitrox) [reply mu-plugins/media-import.php]: @jjgrainger I've added `getenv` for reading from environment variable. We need to merge the .env support PR in order for this to work

File: mu-plugins/media-import.php [hunk 2]
```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * WP-CLI command to import the images from unsplash API.
```
Comment (mukeshpanchal27) [inline mu-plugins/media-import.php]: Can you please add plugin info similar to joe did here https://github.com/mukeshpanchal27/webp-quality-analysis/blob/trunk/mu-plugins/wp-cli-webp-analysis-command.php#L2

**Review by mukeshpanchal27 — APPROVED** [review]

**Pull request merged**